homu/html/queue.html

```diff
@@ -3,6 +3,7 @@
     <head>
         <meta charset="utf-8">
         <title>Homu queue - {{repo_name}}</title>
+        <link href="//cdn.datatables.net/1.10.4/css/jquery.dataTables.min.css">
```

Comment:
Inspecting the HTTP response from their server, it seems that they do provide the `Expires` header, so there wouldn't be much trouble... But I just want to make sure of this.